setup.py

```diff
@@ -1,20 +1,28 @@
 from setuptools import setup, find_packages
 import os
+import sys
 
 repo_base_dir = os.path.abspath(os.path.dirname(__file__))
 
+# import __about__ file from repository to avoid reading from installed package
+sys.path.insert(0, repo_base_dir)
```

Comment:
```suggestion
```

Reply:
I highly recommend to keep the explicit ``sys.path`` management, as otherwise ``__about__`` can be imported from an installed ``tardis``.